.github/actions/environment/action.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build toolchain
+description: Installs Dependencies and Compiles Lingo
+
+runs:
+  using: "composite"
+  steps:
+    - uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+    - uses: actions/setup-java@v3
+      with:
+        distribution: temurin
+        java-version: 17
+    - name: Integration Test
+      run: |
+          # setting java17
+          ls -alh /usr/lib/jvm/
+          # export JAVA_HOME=/usr/lib/jvm/java-17-temurin-amd64
+          
+          # building lingo
+          cargo b --release
+
+          # installing lingua-franca tests           
+          git clone https://github.com/lf-lang/lingua-franca.git ./lingua-franca
+          
+          # installing lfc
+          curl -Ls https://install.lf-lang.org | bash -s cli
```

Comment:
```suggestion
          curl -Ls https://install.lf-lang.org | bash -s nightly cli
```

Reply:
How is it able to pass the tests with the latest release? Changes in the `json` parsing happened later and are only available in `nightly` yet.

Reply:
I changed it to nightly, this also simplified the CI because I don't need to figure out the current version of lfc and can just checkout the nightly tag in the lingua-franca repo

Reply:
=> done